### PR TITLE
Add OpenSearch integration page

### PR DESCRIPTION
**Pull request by mavam**
This PR adds a new integration page for OpenSearch

### CODE REVIEW

File: web/docs/integrations/opensearch/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Instead of treating the entire event as document to be indexed by OpenSearch,
+you can designate a nested record as document:
+
+```tql
+from {category: "qux", doc_id: "XXX", event: {foo: "bar"}}
+to_opensearch "localhost:9200", id=doc_id, doc=event, action="update", index=category
+```
+
+The above example udpates the document with ID 42 with the contents from the
```
Comment (lo-chr) [inline web/docs/integrations/opensearch/README.md:39]: "udpates" -> "updates"
```suggestion
The above example updates the document with ID 42 with the contents from the
```

**Review by lo-chr — CHANGES_REQUESTED** [review]
Just a little typo.

File: web/docs/integrations/opensearch/README.md (hunk 1 above)
Comment (raxyte) [inline web/docs/integrations/opensearch/README.md:39]:
```suggestion
The above example udpates the document with ID `XXX` with the contents from the
```

**Review by dominiklohmann — APPROVED** [review]
I gave this a quick spin with my local OpenSearch instance, and the provided pipelines appear to work as expected.

🙏

**Pull request merged**